udata/core/dataset/api_fields.py

```diff
@@ -160,6 +160,8 @@
 
 dataset_full_fields = api.inherit('DatasetFull', dataset_fields, {
     'quality': fields.Raw(description='The dataset quality', readonly=True),
+    'last_update': fields.ISODateTime(
```

Comment:
I hesitated but given that `last_update` is a computation which iterate on each resource, I eventually chose to put it in the `full` version. Waiting for fields-on-demand to fix that once and for all ;-).